### DISCUSSION TITLE
How many devices can I log in to at the same time?

### DISCUSSION THREAD

**Discussion by youngtofighting** [community/community#17604] [Other Feature Feedback, Questions, & Ideas]
I was logged in on one Xiaomi laptop, but not on the other Mac.

**Comment by MrKrishnaAgarwal** [comment]
as many as you want.
you can open github in laptops, desktops, mobiles, tv's

**Comment by itzkavindu** [comment]
You can login to github from as many devices as you want

**Comment by savanrajkotiya** [comment]
Basically, there is no limit to logged-in devices at the same time, you can logged in to N devices at the same time.

**Comment by Zi-Tao** [comment]
@youngtofighting  No restrictions!
